### ISSUE TITLE
Error occured while creating symlink admin frontend build

### ISSUE THREAD

**Issue by dduenker** [sulu/sulu#7365]
| Q | A
| --- | ---
| Sulu Version | 2.5.14
| PHP Version | 8.3
| DB Version | irrelevant
| Browser Version | irrelevant

# Actual Behavior

Building the admin frontend currently fails, because the command "npx symlink-dir" uses the latest release of symlink-dir.
The latest version of this requires node >= 18.12 now, whereas the sulu frontend requires node 14.

breaking change:  https://github.com/pnpm/symlink-dir/commit/55fd6971af81b0e06d6c3614cfa6564d7835e70c

# Expected Behavior

Building the frontend does not fail

# Steps to Reproduce

Build manually as described on https://docs.sulu.io/en/latest/cookbook/build-admin-frontend.html#common-errors

# Possible Solutions

pin the used symlink-dir to a version before 6.0.0

**Comment by cteliberty** [comment]
This was fixed in #7364, but the test look like it's blocked

**Comment by dduenker** [comment]
Thank you for the info @cteliberty , should i leave this issue open, until the change is available, or should i close it?

**Comment by cteliberty** [comment]
@alexander-schranz Is there a problem with the test? The tests seems no longer progressing.
https://github.com/sulu/sulu/actions/runs/8658836806/job/23743502513

@dduenker I don't know.

**Comment by alexander-schranz** [comment]
A release is planned for this days. Still you can patch the file manually to unblock it:

`vendor/sulu/sulu/symlink-vendor-directory.js`:

```diff
diff --git a/symlink-vendor-directory.js b/symlink-vendor-directory.js
index d33c78010f0..7c502c98d3b 100644
--- a/symlink-vendor-directory.js
+++ b/symlink-vendor-directory.js
@@ -33,7 +33,7 @@ if (
         && path.basename(path.dirname(process.cwd())) === 'assets'
     )
 ) {
-    exec('npx symlink-dir ' + from + ' ' + to, (error) => {
+    exec('npx "symlink-dir@<6.0" ' + from + ' ' + to, (error) => {
         if (error) {
             throw new Error('Error occured while creating symlink: ' + error);
         }
```

Unreleased version of sulu/sulu can also be installed using:

```bash
composer require sulu/sulu:"2.4.*@dev"

composer require sulu/sulu:"2.5.*@dev"
```

**Comment by YetiCGN** [comment]
Thanks! This had me stumped for a while and I was frantically trying to update the admin JS dependencies to Node 18...

**Comment by alexander-schranz** [comment]
Another issue appeared and we decided to downgrade the symlink package in: https://github.com/sulu/sulu/pull/7504
